server/src/server.ts

```diff
@@ -65,6 +65,11 @@ let codeActionsFromDiagnostics: codeActions.filesCodeActions = {};
 // will be properly defined later depending on the mode (stdio/node-rpc)
 let send: (msg: p.Message) => void = (_) => {};
 
+let findBinary = (projectRootPath: p.DocumentUri) =>
```

Comment:
Which binary?
`findBuildBinary`.

Reply:
All these functions, including `findNodeBuildOfProjectRoot` need to be renamed to contain the name `Build`.
So `findBuildBinaryFromProjectRoot` and `findBuildBinaryFromConfig`.